docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,146 @@
+---
+id: learn-async
+title: Asynchronous Backing
+sidebar_label: Asynchronous Backing
+description: A brief overview of asynchronous backing, and how it affects Polkadot's scalability.
+keywords: [parachains, slots, backing, parablock]
+slug: ../learn-async
+---
+
+Asynchronous backing is a mechanism that introduces an enhancement to the block validation pipeline
+for parachain **<->** relay chain communication. This pipeline will allow collators to include more
+transactions/data in parablocks while reducing parablock validation times from around 12 to 6
+seconds. In short, asynchronous backing will speed up the parachain performance multifold.
+
+It has three overarching goals:
+
+1. Decrease parachain blocks (parablock) validation time to **6 seconds** from **12 seconds**
+2. Increase the amount of usable blockspace by a factor of 5-10, meaning more state changes are
+   allowed per relay chain block.
+3. Allow for parablocks to be re-proposed to the network if they are not included initially on the
+   first attempt.
+
+Asynchronous backing works by providing a form of **contextual execution**, which allows for more
+time for parachain collators to fit more transactions and ready block candidates for backing and
+inclusion. **Contextual execution** refers to how a parablock can begin being built earlier through
```

Comment:
The unincluded segment lives in the runtime of the collator, so a candidate is added to the unincluded segment each time the parachain runtime executes a block. 

The relay chain has a more complex structure to track unincluded segments called `ProspectiveParachains`. This structure lives in the relay client, and candidates can only be placed there after at least one backer has cast a `valid` vote for them.

Now for finality. Relay chain finality doesn't care about unincluded segments since only included blocks can be finalized. Parachain finality piggy backs off of relay chain finality and thus doesn't care about unincluded segments either.